### PR TITLE
Feature/degauss widget pdos tab

### PR DESCRIPTION
**Pull request by AndresOrtegaGuerrero**
This tab includes a degauss widget to the pdos plugin
This useful for some case where the PDOS calculation doesnt obtain some projections (is a projwfc,x bug , that Giovanni mentioned) , this is a way around to avoid this. 

image

This solves the issue of core electron density not displayed before for Au 

Now
image

Before 
image

This PR is quite important since it also affects states around fermi level like in this case , if you noticed there are some missing states in the pdos

image

### CODE REVIEW

**Review by cpignedoli — CHANGES_REQUESTED** [review]
The PDOS smearing affects both DOS and PDOS so teh name of teh widget shoudl be different.
-it must be NOT active by default
-It should have a default value that provides similar results compared to default tetraedron parameters (so something close to 0.005 for protocol moderate --I tested Au for this--))

File: src/aiidalab_qe/plugins/pdos/setting.py [hunk 1]
```diff
@@ -31,15 +33,45 @@ def __init__(self, **kwargs):
...
         )
+        self.use_pdos_degauss = ipw.Checkbox(
+            value=False,
+            description="Use custom PDOS degauss",
+            style={"description_width": "initial"},
+        )
+        self.pdos_degauss = ipw.BoundedFloatText(
+            min=0.001,
+            step=0.001,
+            value=0.01,
```
Comment (cpignedoli) [inline src/aiidalab_qe/plugins/pdos/setting.py:44]:
```suggestion
            value=0.005,
```

**Review by cpignedoli — APPROVED** [review]
approved if you merge my new default value

**Review by mikibonacci — APPROVED** [review]
Hi @AndresOrtegaGuerrero, thank you for the PR. 
For me it can be merged, I just left a small comment (nothing relevant actually).

**Pull request merged**